src/sdk/model/entity.ts

```diff
@@ -55,4 +58,40 @@ export abstract class Entity {
    */
   abstract get entityType(): EntityType;
 
+  /**
+   * Returns the api prefix for current entity or null if not defined
+   * @returns {string} the api prefix or null
+   */
+  get apiPrefix(): string | null {
```

Comment:
I really don't like this. If the interface method doesn't apply to every entity then it shouldn't be a method on the interface. It kinda defeats the purpose of having the interface in the first place. I also don't like having this apiPrefix method, its  detail that should be abstracted away by this SDK.